src/ai-dialog-footer.ts

```diff
@@ -27,6 +27,9 @@ export class AiDialogFooter {
   @bindable public buttons: any[] = [];
   @bindable public useDefaultButtons: boolean = false;
 
+  /**
+   * @internal
```

Comment:
I'm OK with that. My reasoning is that this is an API that will be consumed by something external - the DI `Container`. At the same time it is not intended for use by user code.